<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryBookWindow</class>
 <widget class="QMainWindow" name="QueryBookWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="et_bookid">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_bookname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>书名</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_bookid">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>书号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_querybook">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_bookname">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
